azure-functions/acbs-function/mappings/facility/facility-guarantee-amend.js

```diff
@@ -0,0 +1,26 @@
+const facilityGuaranteeAmend = (amendment) => {
+  try {
+    const { amount, facilityGuaranteeDates } = amendment;
+    const record = {};
+
+    if (!amount && !facilityGuaranteeDates) {
+      throw new Error('Either amount or facilityGuaranteeDates.guaranteeExpiryDate is required.');
+    }
+
+    if (amount) {
+      record.guaranteedLimit = amount;
+    }
+
+    if (facilityGuaranteeDates) {
+      record.expirationDate = facilityGuaranteeDates.guaranteeExpiryDate;
+    }
+
+    // Return amended FGR
```

Comment:
What is `FGR` ?

Reply:
This was a suggested comment -- it's matching patterns from the other files -- Facility Guarantee Record